### PR TITLE
fix: add styles override to input text  container

### PR DESCRIPTION
**Pull request by atomictangerine**
## What I did

add a css override prop for the styled container component

### CODE REVIEW

**Review by clairehhelms — APPROVED** [review]

File: ui/input-text/src/InputText.tsx [hunk 1]
```diff
@@ -243,12 +208,31 @@ export const InputText = React.forwardRef<HTMLInputElement, InputTextProps>(
...
+        if (key.includes("padding")) {
+          inputStyles = { ...inputStyles, [key]: css.key };
+          return;
+        }
+        containerStyles = { ...containerStyles, [key]: css.key };
+      });
+    // ----
+
     return (
-      <div>
```
Comment (wp-aberg) [inline ui/input-text/src/InputText.tsx:247]: This div is intended to keep the error message and helper text locked to the input, removing it messes with the spacing of that text relative to the input.

helper-text

error-text

Comment (atomictangerine) [reply ui/input-text/src/InputText.tsx:247]: ah okay okay. That makes sense. Thanks for catching that.

File: ui/input-text/src/InputText.tsx [hunk 2]
```diff
@@ -243,12 +208,31 @@ export const InputText = React.forwardRef<HTMLInputElement, InputTextProps>(
...
+    // This code block because we want to filter out the padding elements and pass those
+    // only to the input element. All other styles should go into the StyledContainer.
+    // If we ever need to check for more attributes, then we should pull this out into a function.
+    let inputStyles = {};
+    let containerStyles = {};
+
+    css &&
+      Object.keys(css).map((key) => {
+        if (key.includes("padding")) {
+          inputStyles = { ...inputStyles, [key]: css.key };
```
Comment (wp-aberg) [inline ui/input-text/src/InputText.tsx:221]: testing this out, needs to be square brackets around the key
```javascript
inputStyles = { ...inputStyles, [key]: css[key] };
```

File: ui/input-text/src/InputText.tsx [hunk 3]
```diff
@@ -243,12 +208,31 @@ export const InputText = React.forwardRef<HTMLInputElement, InputTextProps>(
...
+          return;
+        }
+        containerStyles = { ...containerStyles, [key]: css.key };
```
Comment (wp-aberg) [inline ui/input-text/src/InputText.tsx:224]:
```javascript 
containerStyles = { ...containerStyles, [key]: css[key] };
```

Comment (atomictangerine) [reply ui/input-text/src/InputText.tsx:224]: omg! good catch! updating now

**Review by wp-aberg — APPROVED** [review]
Looks good, all set.

**Pull request merged**